RELEASE_NOTES.md

```diff
@@ -1,27 +1,2 @@
-#### 1.2.3 July 10 2017

```

Comment:
@ayoung would you mind keeping the old release notes and appending to them in this instance?